### PR TITLE
delete status from pool, while deleting child VS

### PR DESCRIPTION
**Pull request by chauhanshubham**

### CODE REVIEW

**Review by monotosh-avi — APPROVED** [review]

**Review by akshayhavile — APPROVED** [review]
LGTM with one minor nit.

File: internal/nodes/dequeue_ingestion.go [hunk 1]
```diff
@@ -419,6 +419,7 @@ func handleL4Service(key string, fullsync bool) {
 					Namespace: namespace,
 					Key:       key,
 				}
+				utils.AviLog.Infof("PublishToStatusQueue KEYL: %s, OPTION: %v", name, utils.Stringify(statusOption))
```
Comment (sudswasavi) [inline internal/nodes/dequeue_ingestion.go:422]: KEYL is intentional?

**Review by sudswasavi — COMMENTED** [review]
I see this PR clogging the logs with a lot these publishtostatusque ones. Is that intentional?

**Comment by chauhanshubham** [comment]
Please don't merge this @sudswasavi @akshayhavile @monotosh-avi This is just for our internal reference. It indicates the changes provided internally for testing. The logs are intentional as of now and will be removed later. Thanks

**Comment by chauhanshubham** [comment]
> Please don't merge this @sudswasavi @akshayhavile @monotosh-avi This is just for our internal reference. It indicates the changes provided internally for testing. The logs are intentional as of now and will be removed later. Thanks

Sanitised the logs, good to go now

**Review by akshayhavile — APPROVED** [review]

**Review by sudswasavi — APPROVED** [review]

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by sudswasavi** [comment]
build ako

**Comment by sudswasavi** [comment]
build AKOs

**Comment by sudswasavi** [comment]
build ako

**Pull request merged**